### DISCUSSION TITLE
queued for download

### DISCUSSION THREAD

**Discussion by Aleksei789** [transmission/transmission#2925] [Q&A]
hello!

I have a very good dedicated server.
I download old content and seeders are distributed there at a very low speed.
I think the transmission has a maximum download of up to 6-8 downloads. everyone else is waiting in line sometimes for two or three days :(

how to increase the number of active downloads to 20? or even 30?
I didn't find this feature in the settings.

I will be grateful for any information.
thanks

**Comment by Zabriskije** [comment]
Preferences → Transfers → Management
Download with maximum of: `number` active transfers
Seeding with maximun of: `number` active transfers

**Comment by Aleksei789** [comment]
> Preferences → Transfers → Management Download with maximum of: `number` active transfers Seeding with maximun of: `number` active transfers

hello

thanks you answer
i use web form in my debian
i not have it's

maybe I need to install some module?


![image](https://user-images.githubusercontent.com/98034927/163557397-557c076a-a5a2-4741-87ad-10fe9d0bf181.png)
![image](https://user-images.githubusercontent.com/98034927/163557433-f2df239e-8823-48ff-8ee8-818804185d34.png)
![image](https://user-images.githubusercontent.com/98034927/163557467-87416784-2c61-4d42-99c6-dacd80d78723.png)
![image](https://user-images.githubusercontent.com/98034927/163557493-1d76f532-8442-4303-b81e-546f4389fe76.png)

**Reply by Zabriskije** [reply]
I don't know about that, sorry.
